jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -876,3 +875,30 @@ def _viewer_is_image_viewer(self):
 
         return isinstance(self.viewer.selected_obj, (ImvizImageView, CubevizImageView,
                                                      MosvizImageView, MosvizProfile2DView))
+
+    @observe('line_color_value', 'image_color_value')
+    def color_change_event(self, event):
+        # Do not run when Plot Options is being instantiated
+        if event['old'] is None:
+            return
+        layer_names = [x['label'] for x in self.layer_items]
+
+        if isinstance(self.layer_selected, list):
+            layers_select = self.layer_selected
+        else:
+            layers_select = [self.layer_selected]
+        for layer_select in layers_select:
+            self.layer_items[layer_names.index(layer_select)]['color'] = event['new']
+        self.send_state('layer_items')

```

Comment:
is this needed (and if so, why) or not anymore now that the mixed logic has moved into `LayerSelect`?